manifests/init.pp

```diff
@@ -175,24 +177,19 @@
     validate_re($logdir,'^(\$(?!/)|/).+')
     validate_re($rundir,'^(\$(?!/)|/).+')
 
+    if $ca_crl_pull_interval != 2 {
+      deprecation('pupmod::ca_crl_pull_interval', 'pupmod::ca_crl_pull_interval is deprecated, the CRL cron job has been removed.')
```

Comment:
Should we change the default interval to {{undef}} and make this more clear?

Since this actually has the potential to break systems, we really want it to be off.